examples/practice-typescript/src/components/TableBody/index.tsx

```diff
@@ -0,0 +1,30 @@
+import TableRow from 'components/TableRow';
+import User from 'scripts/types/user';
+
+interface TableListProps {
+  children: User[];
+}
+
+const editUser = () => console.log('edit');
+
+const deleteUser = () => console.log('delete');
+
+const TableBody = ({ children = [] }: TableListProps) => {
+  return (
+    <tbody>
+      {children.map(({ id, email, firstName, lastName, phoneNumber }) => (
+        <TableRow
+          id={id}
```

Comment:
missing key, please check if ur eslint work or not

Reply:
Key in tableRow

Reply:
Fixed